EIPS/eip-4973.md

```diff
@@ -0,0 +1,127 @@
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+`ERC-4973` tokens _must_ implement the interfaces:
+
+- [ERC-165](./eip-165.md)'s `ERC165` (`0x01ffc9a7`)
+- [ERC-721](./eip-721.md)'s `ERC721Metadata` (`0x5b5e139f`)
+
+`ERC-4973` tokens _must not_ implement the interfaces:
+
+- [ERC-721](./eip-721.md)'s `ERC721` (`0x80ac58cd`)
+
+```solidity
+pragma solidity ^0.8.6;
+
+/// @title Non-Transferrable Non-Fungible Account-bound Tokens (formerly Soulbound Tokens or "Badges")
+/// @dev See https://eips.ethereum.org/EIPS/eip-4973
+///  Note: the ERC-165 identifier for this interface is 0x6352211e.
+interface IERC4973 /* is ERC165, ERC721Metadata */ {
+  /// @dev This emits when transfer of any account-bound token is established by
+  /// any mechanism. This event emits when ACTs are created (`from` == 0) and
+  /// destroyed or revoked (`to` == 0). Exception: during contract creation, any
+  /// number of ACTs may be created and assigned without emitting Transfer.
+  event Transfer(address indexed _from, address indexed _to, uint256 indexed _tokenId);
+  /// @notice Find the address bound to an ERC4973 account-bound token
+  /// @dev ACTs assigned to zero address are considered invalid, and queries
+  ///  about them do throw.
+  /// @param _tokenId The identifier for an ACT
+  /// @return The address of the owner bound to the ACT
+  function ownerOf(uint256 _tokenId) external view returns (address);
+}
+
+interface ERC165 {
+  /// @notice Query if a contract implements an interface
+  /// @param interfaceID The interface identifier, as specified in ERC-165
+  /// @dev Interface identification is specified in ERC-165. This function
+  ///  uses less than 30,000 gas.
+  /// @return `true` if the contract implements `interfaceID` and
+  ///  `interfaceID` is not 0xffffffff, `false` otherwise
+  function supportsInterface(bytes4 interfaceID) external view returns (bool);
+}
+
+/// @title ERC-721 Non-Fungible Token Standard, optional metadata extension
+/// @dev See https://eips.ethereum.org/EIPS/eip-721
+///  Note: the ERC-165 identifier for this interface is 0x5b5e139f.
+interface ERC721Metadata /* is ERC721 */ {
+  /// @notice A descriptive name for a collection of NFTs in this contract
+  function name() external view returns (string _name);
+
+  /// @notice An abbreviated name for NFTs in this contract
+  function symbol() external view returns (string _symbol);
+
+  /// @notice A distinct Uniform Resource Identifier (URI) for a given asset.
+  /// @dev Throws if `_tokenId` is not a valid NFT. URIs are defined in RFC
+  ///  3986. The URI may point to a JSON file that conforms to the "ERC721
+  ///  Metadata JSON Schema".
+  function tokenURI(uint256 _tokenId) external view returns (string);
+}
```

Comment:
You can just link to these (eg. `[EIP-165](./eip-165.md)`), or leave them here. Just feels a tad redundant.

Reply:
I feel it reads better when those snippets are included. Is this a hard objection on your side or merely a styling suggestion because if I can overrule, I'd like to leave them thanks.

Reply:
Please remove these, you should just link to EIPs that are "required" for your EIP.